crates/ruma-federation-api/src/thirdparty/bind_callback.rs

```diff
@@ -90,8 +90,9 @@ pub mod v1 {
         /// The user ID that sent the invite.
         pub sender: OwnedUserId,
 
-        /// Signature from the identity server using a long-term private key.
-        pub signed: BTreeMap<OwnedServerName, BTreeMap<OwnedServerSigningKeyId, String>>,
+        /// A block of content which has been signed, which servers can use to verify the
+        /// third-party invite.
+        pub signed: SignedContent,
```

Comment:
I am also wondering if we should always wrap `SignedContent` in `Raw`. It would facilitate the conversion to `CanonicalJsonObject` to check the signatures.

Reply:
Done.